app/javascript/packages/form-steps/form-steps.spec.tsx

```diff
@@ -19,11 +20,13 @@ interface StepValues {
 
 describe('FormSteps', () => {
   const sandbox = sinon.createSandbox();
+  sinon.spy(analytics, 'trackEvent');
```

Comment:
@aduth  I would get errors every time I ran the test suite if the` trackEvent` spy was in each test so I moved it to the top of this file, but not sure if there is a better place for this to live.

Reply:
Can we at least put it in a `beforeEach`, so that we don't have to worry about side-effects related to the spy due to the test being loaded, but before it's run?

Also, we should use the `sandbox` object here to define the spy, which is already set up to restore the original function implementation to ensure the spy doesn't linger beyond the test.

```suggestion

  beforeEach(() => {
    sandbox.spy(analytics, 'trackEvent');
  });
```

Reply:
When I write it this way I get a type error... `TypeError: [Function: trackEvent] is not a spy or a call to a spy!`

Reply:
I see it's erroring in the specs for PersonalKeyStep and PersonalKeyConfirmStep. The specs are isolated, so we'll need to have the spy in each spec file.